tidy3d/plugins/invdes/design.py

```diff
@@ -292,11 +305,10 @@ def to_simulation(self, params: anp.ndarray) -> dict[str, td.Simulation]:
         simulation_list = [design.to_simulation(params) for design in self.designs]
         return dict(zip(self.task_names, simulation_list))
 
-    def to_simulation_data(self, params: anp.ndarray, **kwargs) -> web.BatchData:
+    def to_simulation_data(self, params: anp.ndarray, **kwargs):  # -> "web.BatchData"
```

Comment:
I was a bit confused about this, I get an error if I uncomment (and also ruff complains), do we have a good way to do type annotation without importing? I remember there was something.

Reply:
maybe just put it it as a string `'web.BatchData`.

Reply:
that's strange, `from __future__ import annotations` should have exactly the same effect as wrapping the type hints in strings in python 3.9, and above it shouldn't even be necessary. what python version were you testing this with @momchil-flex?

Reply:
could it be that you were just seeing a lint error? probably ruff doesn't properly recognize this properly as a forward reference. I checked out your branch and uncommented and it's working minus the lint error, which in this case should just be disabled, so the following works for me:
```python
def run_async(self, simulations, **kwargs) -> web.BatchData:  # noqa: F821
    ...
```

Reply:
Yeah this does seem to work.